app/server/config.ts

```diff
@@ -1,4 +1,10 @@
+declare var DOMAIN: string;
```

Comment:
what's the purpose of these outer vars?

Reply:
this tells teamcity that these are valid globals